src/model/documents/ingredient.ts

```diff
@@ -0,0 +1,11 @@
+import { BaseModel } from '../base-model';
+import { UnitOfMeasure } from '../enums/unit-of-measure';
+import { Category } from './category';
+
+export interface Ingredient extends BaseModel {
+  imagePath: string;
+  name: string;
+  category: Category;
+  unitOfMeasure: UnitOfMeasure;
+  supplies: number;
```

Comment:
How is this property supposed to work?

Reply:
It is initially defined when creating the ingredient and then replenished by finished deliveries.
From what I remember we discussed it that way.